doc/adrs/168_add_support_for_forking_table_description_from_representation.adoc

```diff
@@ -0,0 +1,48 @@
+= ADR-166 Add support for forking table description from representation
+
+== Context
+
+On a table representation, we want to be able to modify the specification used as description of this table and then reload the table with this new specification.
+
+== Decision
+
+We will allow users to open a new studio from an opened table representation. This new studio will be a fork of the one used to instantiate the table. After modifying this new studio, the user will be able to re open the previous table representation that will use the updated studio as description.
```

Comment:
- one sentence per line please :)
- "This new studio will be a fork of the one used to instantiate the table."  I would say that the new studio will contain a copy of the table description used to instantiate the table. Technically, in the first use case, the original table description may be contributed programmatically and created with the view builder API. It may not come from a "studio"
- "that will use the updated studio as description" -> that will use the updated table description

Reply:
In that case the datafetcher will not have enought informations to do everything.
I need an event handler that have access to the current editing context in order to get the original table description.